Jenkinsfile

```diff
@@ -191,17 +262,26 @@ pipeline {
 	}
 	post {
 		always {
-			// teardown() should have run cleanWs()
+			// teardown_mocha() should have run cleanWs()
```

Comment:
Why is this commented out?

Reply:
I think it's just a description for developers.